<launch>

  <node name="arm_moveit_kinematics_node" pkg="arm_moveit_kinematics" type="arm_moveit_kinematics_node" respawn="true" output="screen">
  <param name="poses_yaml" type="string" value="$(find arm_moveit_kinematics)/config/poses.yaml"/>
  <param name="stl_path" type="string" value="package://arm_moveit_kinematics/resource/"/>
  </node>
  <node name="arm_keyboard_node" pkg="arm_moveit_kinematics" type="arm_keyboard_node" respawn="false" output="screen" launch-prefix="gnome-terminal -x">
  </node>
  <!--node name="arm_joints_controller_node" pkg="arm_moveit_kinematics" type="arm_joints_controller_node" respawn="false" output="screen">
  </node-->
  
  <!--ode name="mineral_detect_node" pkg="mineral_detect" type="mineral_detect_node" respawn="false" output="screen">
  </node-->
  
  <node name="arm_3d_perception" pkg="arm_moveit_kinematics" type="arm_3d_perception" respawn="false" output="screen">
  </node>

  <!--node pkg="rosbag" type="record" name="bag_record" args="-O godzilla.bag -a" /-->

</launch>